code/game/objects/items/weapons/power_cells.dm

```diff
@@ -1,4 +1,7 @@
 //BIG CELLS - for APC, borgs and machinery.
+/obj/item/weapon/cell
+
```

Comment:
That's a really wrong way to do it. Never declare object somewhere else just to add some vars to it.